<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeDetectWindow</class>
 <widget class="QMainWindow" name="EdgeDetectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Sobel">
     <attribute name="title">
      <string>Sobel</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sobel Filter</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="sobelImage1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="SelectButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="sobelImage2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="text3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="text2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Value</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Canny">
     <attribute name="title">
      <string>Canny</string>
     </attribute>
     <widget class="QLabel" name="cannyImage1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cannyImage2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cany Edge Detection</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="SelectButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="text3_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="text2_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold 2</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Laplacian">
     <attribute name="title">
      <string>Laplacian</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Laplacian operator</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="laplaceButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="laplaceImage2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="laplaceImage1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
